client/package.json

```diff
@@ -21,11 +22,13 @@
     "@types/react": "^18.0.21",
     "@types/react-dom": "^18.0.6",
     "axios": "^1.1.3",
+    "firebase": "^9.12.1",
     "font-awesome": "^4.7.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-router": "^6.4.0",
-    "react-router-dom": "^6.4.2",
+    "react-hook-form": "^7.37.0",
+    "react-router-dom": "^6.4",
     "react-scripts": "5.0.1",
     "react-toastify": "^9.0.8",
     "styled-components": "^5.3.6",
```

Comment:
@AseelL @mohmmed23 @Saeed99Madi @salsabeelomar Why do we have styled components installed when we are using MUI and emotion?